docs/application/dotnet/guides/personal/account.md

```diff
@@ -463,11 +464,11 @@ The following table lists the account properties that can be modified.
 
 | Account property    | Data type                          | Mandatory | Description                              |
 |-------------------|----------------------------------|---------|----------------------------------------|
-| User name           | `String`                           | Yes       | Identity of an account.If the display name and email ID are not set for an account, the user name is shown for the account on the Accounts screen in the Setting application. |
+| User name           | `String`                           | Yes       | Identity of an account.<br>If the display name and email ID are not set for an account, the user name is shown for the account on the Accounts screen in the Setting application. |
```

Comment:
Remove code tag(\'  \') in Data type column. Apply all item in Data type column.